main.tf

```diff
@@ -12,17 +12,27 @@ locals {
   ]
 
   website_endpoints = [
-    "${aws_s3_bucket.redirect.website_endpoint}",
-    "${aws_s3_bucket.main.website_endpoint}",
+    "${aws_s3_bucket.redirect.bucket_domain_name}",
+    "${aws_s3_bucket.main.bucket_domain_name}",
   ]
 }
 
 data "aws_route53_zone" "zone" {
   name = "${var.domain}"
 }
 
+data "template_file" "bucket_policy" {
+  template = "${file("${path.module}/website_bucket_policy.json")}"
```

Comment:
Have you considered using an [iam_policy_document](https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html) instead?

Reply:
Yes, I have looked at this, but it seemed easier for me to use a template file since I could then just copy-paste the JSON which I already had in my bucket, that was pre-filled for me by CloudFront when setting things up manually - and customize that using interpolation.

Do you feel strongly about changing it or are you OK leaving this as it is?

Reply:
I think I can live with that. :) I might change it myself in the future.